src/Service/Index.php

```diff
@@ -85,14 +87,61 @@ public function indexCourses(array $courses) : bool
     public function deleteCourse(int $id) : bool
     {
         $result = $this->delete([
-            'index' => Search::PUBLIC_INDEX,
+            'index' => Search::COURSE_INDEX,
             'type' => CourseDTO::class,
             'id' => $id,
         ]);
 
         return !$result['errors'];
     }
 
+    /**
+     * @param Descriptor[] $descriptors
+     * @return bool
+     */
+    public function indexMeshDescriptors(array $descriptors) : bool
+    {
+        foreach ($descriptors as $descriptor) {
```

Comment:
under which circumstances would this happen?

Reply:
Probably never, but I feel like it would be confusing if we sent DTOs here and it broke weirdly. The same check exists for User/Course because I was worried we would send an entity and not a DTO at some future point.

Reply:
fair enough.